stdlib/gc.mli

```diff
@@ -439,6 +439,13 @@ external eventlog_resume : unit -> unit = "caml_eventlog_resume"
    was started with OCAML_EVENTLOG_ENABLED=p. (which pauses the collection of
    traces before the first event.) *)
 
+type event
+
+external new_event : string -> event = "caml_eventlog_new_event"
+
+external emit_begin_event : event -> unit = "caml_eventlog_emit_begin_event"
+
+external emit_end_event : event -> unit = "caml_eventlog_emit_end_event"
```

Comment:
In the mli file, you should write these as e.g.:

    val new_event : string -> event

which hides the fact that it's implemented as a C function named `caml_eventlog_new_event`, and means the interface doesn't have to change if the C function ever gets renamed / moved to OCaml code.